api/handlers/container/start.go

```diff
@@ -17,7 +19,26 @@ import (
 func (h *handler) start(w http.ResponseWriter, r *http.Request) {
 	cid := mux.Vars(r)["id"]
 	ctx := namespaces.WithNamespace(r.Context(), h.Config.Namespace)
-	err := h.service.Start(ctx, cid)
+
+	detachKeys := r.URL.Query().Get("detachKeys")
+
+	devNull, err := os.OpenFile("/dev/null", os.O_WRONLY, 0600)
+	if err != nil {
+		response.JSON(w, http.StatusBadRequest, response.NewErrorFromMsg("failed to open /dev/null"))
+		return
+	}
+	defer devNull.Close()
+
+	containers := [1]string{cid}
+	globalOpt := ncTypes.GlobalCommandOptions(*h.Config)
+	options := ncTypes.ContainerStartOptions{
+		GOptions:   globalOpt,
+		Stdout:     devNull,
+		Attach:     false,
```

Comment:
nerdctl `start` supports `attach` https://github.com/containerd/nerdctl/blob/main/docs/command-reference.md#whale-nerdctl-start. Can we pass it, instead of setting this to False by default.

Reply:
Would that be an additional option in container start API? 
Docker doesn't support it at the moment - https://docs.docker.com/reference/api/engine/version/v1.43/#tag/Container/operation/ContainerStart

Reply:
I see. Looks like docker-cli makes a separate ContainerAttach call for that https://github.com/docker/cli/blob/master/cli/command/container/start.go#L113. 

The current implementation looks good to me, but we should follow up later on the end to end Start with attach behavior